fv3config/_datastore.py

```diff
@@ -166,5 +168,47 @@ def extract_data():
         shutil.move(os.path.join(app_data_dir, filename_root), local_archive_dir)
 
 
+def is_gsbucket_url(path):
+    if path.startswith(gs_bucket_prefix):
+        return True
+    else:
+        return False
+
+
+def resolve_option(option, built_in_options_dict):
+    """Determine whether a configuration dictionary option is a built-in option or
+    not and return path to file or directory representing option. An option is
+    assumed to be built-in if it is not an absolute path and does not begin with gs://
+
+    Args:
+        option (str): an option
+        built_in_options_dict (dict): built-in options
+
+    Returns:
+        (str): a path or url
+
+    Raises:
+        ConfigError: if option is an absolute path but does not exist or if
+                     option is not in default_options_dict
+    """
+    if os.path.isabs(option):
```

Comment:
Perhaps add a comment mentioning that this nested if statement is a reflection of FV3's internal logic, not a design choice on our part.

Reply:
Good idea. Added.